WapTenantPublicAPI.psm1

```diff
@@ -549,6 +549,192 @@ function GetWAPSubscriptionQuota {
     
 }
 
+function Get-WAPSubscriptionQuota{
+    <#
+    .SYNOPSIS
+        Retrieves a VM Role Quota information object (WAP.Quota).
+
+    .PARAMETER Subscription
+        The Subscription(s) you wish to collect quota information for.
+
+    .EXAMPLE
+        PS C:\>$URL = 'https://publictenantapi.mydomain.com'
+        PS C:\>$creds = Get-Credential
+        PS C:\>Get-WAPToken -Credential $creds -URL 'https://sts.adfs.com' -ADFS
+        PS C:\>Connect-WAPAPI -URL $URL
+        PS C:\>$Sub = Get-WAPSubscription -Name MySubscription 
+        PS C:\>Get-WAPSubscriptionQuota -Subscription $Sub
+    #>
+[CmdletBinding()]
+Param(
+    [Parameter(
+        Mandatory=$False,
+        ValueFromPipeline=$True,
+        ValueFromPipelineByPropertyName=$True,
+        Position=0
+    )]
+    [System.Management.Automation.PSTypeName('WAP.Subscription')] 
+    $Subscription = (Get-WAPSubscription)
+)
+process{
+    ForEach ($Sub in $Subscription){
+        Select-WAPSubscription -Subscription $Sub
+        $VMs = Get-WAPVM 
+        $Cores = $Vms.CpuCount | Measure-Object -Sum 
+        $Memory = $Vms.Memory | Measure-Object -Sum 
+
+        $Services = $Sub.Services | Select BaseQuotaSettings
+
+        ForEach($Service in $Services){
+            $BaseQuotaSettings = $Service.BaseQuotaSettings
+            ForEach($Setting in $BaseQuotaSettings){ 
+                if($Setting.Key -eq "Clouds"){
+                    [xml]$Data = $Setting.Value 
+                    $Output = $Data.Clouds.Cloud.Quota
+                    $Object = New-Object -Type PsObject -Property @{
+                    SubscriptionName=$Sub.SubscriptionName;
```

Comment:
Indent one level. You can replace the new-object by the [pscustomobject] type accelerator